### PR TITLE
zh-translation:content/zh/news/security/istio-security-2019-006/index.md

### PR DESCRIPTION
**Pull request by kylesliu**
ref: https://github.com/servicemesher/istio-official-translation/issues/1601

### CODE REVIEW

File: content/zh/news/security/istio-security-2019-006/index.md [hunk 1]
```diff
@@ -16,24 +16,24 @@ aliases:
         vector="CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:H/RL:O/RC:C"
         releases="1.3 to 1.3.4" >}}
 
-## Context
+## 内容{#context}
 
-Envoy, and subsequently Istio, are vulnerable to the following DoS attack.
-An infinite loop can be triggered in Envoy if the option `continue_on_listener_filters_timeout` is set to `True`. This has been the case for Istio since the introduction of the Protocol Detection feature in Istio 1.3
-A remote attacker may trivially trigger that vulnerability, effectively exhausting Envoy’s CPU resources and causing a denial-of-service attack.
+Envoy 以及随后的 Istio 容易受到以下DoS攻击。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-006/index.md]: DoS 两边需要空格

File: content/zh/news/security/istio-security-2019-006/index.md [hunk 2]
```diff
@@ -16,24 +16,24 @@ aliases:
...
+## 影响范围{#impact-and-detection}
 
-Both Istio gateways and sidecars are vulnerable to this issue. If you are running one of the affected releases, your cluster is vulnerable.
+Istio gateway 和 sidecar 都容易受到此问题的影响。如果您运行的 Istio 是受影响的发行版本，那么您的集群容易受到攻击。
 
-## Mitigation
+## 防范{#mitigation}
 
-* Workaround: The exploitation of that vulnerability can be prevented by customizing Istio installation (as described in [installation options](/zh/docs/reference/config/installation-options/#pilot-options) ), using Helm to override the following options:
+* 解决方法: 通过自定义安装 Istio 可以防止对该漏洞的利用(as described in [installation options](/zh/docs/reference/config/installation-options/#pilot-options))，在使用 Helm 时添加以下选项:
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-006/index.md]: as described in [installation options](/zh/docs/reference/config/installation-options/#pilot-options)
未译

Comment (kylesliu) [reply content/zh/news/security/istio-security-2019-006/index.md]: 已经修改，请review

**Comment by tanjunchen** [comment]
/lgtm
@rootsongjc

**Pull request merged**